README.md

```diff
@@ -30,15 +30,17 @@ Guidance around NGINX Plus is available [here](https://www.nginx.com/blog/deploy
 Dockerfiles contained in this repository are supported by and tested against NGINX Controller version 3.10 and later.
 Note: When building NGINX Plus into a container, be sure to remove repository credentials from your container.
 
-### 1.1. NGINX Controller Agent Inside Docker Container
+### 1.1. Current Limitations
 
-The Controller Agent can be deployed in a Docker environment to monitor and/or configure NGINX processes inside Docker containers.
-The Controller Agent can collect most of the metrics.
+The following list summarizes the existing limitations of monitoring containers with NGINX Controller:
+
+- The Controller Agent can only monitor NGINX from inside the container. It is not currently possible to run the Controller Agent in a separate container and monitor the neighboring containers running NGINX.
 
-The "agent-inside-the-container" is currently the only mode of operation. In other words, the Controller Agent should be running in the same container as the NGINX process being managed/monitored.
-For more information, please refer to our [Controller Dockerfile repository](https://github.com/nginxinc/docker-nginx-controller.git).
+### 1.2. NGINX Controller Agent inside Docker container
 
-### 1.2. Standalone Mode
+The Controller Agent can be deployed in a Docker environment to monitor and/or configure NGINX processes inside Docker containers. The Controller Agent can collect most of the metrics.
+
+### 1.3. Standalone Mode
```

Comment:
This is about runtime - it should be merged with the runtime section.
With Controller 3.x all instances are 'stand-alone' - this is legacy language from the Amplify history that we need to remove.

Containers have build time guidance and run-time guidance.